e2e/t/exclude-expressions-npar.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: tenet.cybozu.io/v1beta2
+kind: NetworkPolicyAdmissionRule
+metadata:
+    name: exclude-only-npar
+spec:
+    namespaceSelector:
+      excludeLabelExpressions:
+      - key: team
+        operator: In
+        values:
+        - neco
```

Comment:
Is it ok that we don't check to be able to exclude multiple values?

Reply:
I thought it would be enough just to use an expression.
But, as you say, let's check that it works properly with multiple values to confirm it more accurately.

Reply:
I appended a value to that manifest and updated the test for expressions.